Contents/Sketch/typography.js

```diff
@@ -7,7 +7,10 @@ var applyTypography = function (newStyles, sharedStyles) {
     'justified': 3
   };
 
-  removeAllStyles();
```

Comment:
If we're not going to call this, we should just remove the function entirely. And also update the README to indicate that styles will be duplicated every time the plugin is run.